tools/protodoc/protodoc.bzl

```diff
@@ -64,10 +64,11 @@ def _proto_doc_aspect_impl(target, ctx):
     args += ["-I" + import_path for import_path in import_paths]
     args += ["--plugin=protoc-gen-protodoc=" + ctx.executable._protodoc.path, "--protodoc_out=" + output_path]
     args += [_proto_path(src) for src in target[ProtoInfo].direct_sources]
-    ctx.action(
+    ctx.actions.run(
         executable = ctx.executable._protoc,
         arguments = args,
-        inputs = [ctx.executable._protodoc] + target[ProtoInfo].transitive_sources.to_list(),
+        inputs = target[ProtoInfo].transitive_sources.to_list(),
```

Comment:
you might not need the `.to_list()` here, seems like you should be able to pass a depset just fine

Reply:
yeah that works, thanks!